navidrome/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+services:
+  navidrome:
+    image: deluan/navidrome:latest
+    restart: on-failure
+    environment:
+      - PUID=1000
+      - PGID=1000
+    ports:
+      - "4533:4533"
+    volumes:
+      - ${APP_DATA_DIR}/data:/data
+      - ${UMBREL_ROOT}/data/storage/music:/music:rw
```

Comment:
Please create a file called `navidrome/data/.gitkeep`.
The problem is that on older versions of umbrelOS the permissions of automatically created repositories are wrong. By supplying a .gitkeep file, those directories are getting created by installing the app automatically.